docs/Configuration/configuration-files/README.md

```diff
@@ -1233,11 +1219,11 @@ If you're using Fleet Premium, these profiles apply to all hosts assigned to no
     macos_settings:
       custom_settings:
         - path: '/path/to/profile1.mobileconfig'
-          labels:
+          labels_include_all:
```

Comment:
Add a second name for existing `labels` key: `labels_include_all`

Both keys have the same behavior: profile is only applied to hosts that have all labels.